examples/mongoose.test.js

```diff
@@ -1055,6 +1055,24 @@ describe('Mongoose', function() {
       // acquit:ignore:end
       // Connect to a MongoDB server running on 'localhost:27017' and use the
       // 'test' database.
+      console.log(mongoose.connection.readyState); //logs 0
```

Comment:
I'd like these changes to be in a separate test. This test is already used as an example in another tutorial `connect.md`, and I'd rather not have these changes affect the other tutorial.